sound/soc/codecs/Kconfig

```diff
@@ -1185,10 +1185,14 @@ config SND_SOC_RT700_SDW
 config SND_SOC_RT711
 	tristate
 
+config SND_SOC_RT711_SDCA
+	tristate
+
 config SND_SOC_RT711_SDW
 	tristate "Realtek RT711 Codec - SDW"
 	depends on SOUNDWIRE
 	select SND_SOC_RT711
+	select SND_SOC_RT711_SDCA
```

Comment:
Does the RT711-SDCA support an HDaudio mode?
I would err on the side of creating a different codec, e.g. SND_SOC_RT711_SDCA_SDW, since in the Makefile you don't actually refer to rt711.c

Reply:
> Does the RT711-SDCA support an HDaudio mode?
> I would err on the side of creating a different codec, e.g. SND_SOC_RT711_SDCA_SDW, since in the Makefile you don't actually refer to rt711.c

RT711-SDCA only supports the Soundwire interface. 
Thanks for your comments and I checked all. I will send the third version later.